lib/honeybadger.ex

```diff
@@ -209,15 +210,34 @@ defmodule Honeybadger do
     |> Client.send_notice()
   end
 
+  @doc """
+  Retrieves the context that will get sent to the Honeybadger API when/if an
+  exception occurs in the current process.
+  """
+  @spec context :: context
   def context do
     (Process.get(@context) || %{}) |> Enum.into(Map.new)
```

Comment:
I think we can use `Process.get/2` here, to set a default to `%{}` without using `||`. The value can't ever be set as `nil` because `context/1` would fail.

Additionally, it would be nice to remove the useless `Enum.into/2`.